SPECS/libcxx/libcxx.spec

```diff
@@ -0,0 +1,568 @@
+%global toolchain clang
+
+# Opt out of https://fedoraproject.org/wiki/Changes/fno-omit-frame-pointer
+# https://bugzilla.redhat.com/show_bug.cgi?id=2158587
+%undefine _include_frame_pointers
+
+%global maj_ver 18
```

Comment:
Let's also tie this macro and the `Version` tag together.

Reply:
Sure thing; updated